ReadyPlayerOne/Controllers/HomeController.cs

```diff
@@ -52,15 +52,15 @@ public IActionResult Index(PlayerViewModel model)
         }
 
         [HttpGet]
-        public IActionResult Details(string id)
+        public IActionResult Details(int id)
```

Comment:
This can be cast as an `int` since the value you pass in will be a number

Reply:
Ah, yes that makes sense! Thank you for catching this.